.github/workflows/ci.yml

```diff
@@ -237,6 +238,23 @@ jobs:
       binary: generate_isograph_fixtures
       params: --dir ./crates/isograph_lang_parser/fixtures/
 
+  build-swc:
+    name: Build swc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
```

Comment:
looks like we may need to install musl-tools: https://github.com/actions-rs/toolchain/issues/102#issuecomment-1132620977 https://github.com/isographlabs/isograph/actions/runs/14565046390/job/41208674999?pr=488